applications/RANSApplication/custom_strategies/algebraic_flux_corrected_scalar_steady_scheme.h

```diff
@@ -0,0 +1,567 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:  Suneth Warnakulasuriya
+//                 Michael Andre, https://github.com/msandre
+//
+
+#if !defined(KRATOS_ALGEBRAIC_FLUX_CORRECTED_SCALAR_STEADY_SCHEME)
+#define KRATOS_ALGEBRAIC_FLUX_CORRECTED_SCALAR_STEADY_SCHEME
+
+// Project includes
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "solving_strategies/schemes/scheme.h"
+#include "utilities/openmp_utils.h"
+#include "utilities/variable_utils.h"
+
+// Application includes
+#include "custom_strategies/relaxed_dof_updater.h"
+#include "rans_application_variables.h"
+
+namespace Kratos
+{
+///@name Kratos Classes
+///@{
+
+template <class TSparseSpace, class TDenseSpace>
+class AlgebraicFluxCorrectedScalarSteadyScheme : public Scheme<TSparseSpace, TDenseSpace>
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    KRATOS_CLASS_POINTER_DEFINITION(AlgebraicFluxCorrectedScalarSteadyScheme);
+
+    using BaseType = Scheme<TSparseSpace, TDenseSpace>;
+
+    using DofsArrayType = typename BaseType::DofsArrayType;
+
+    using TSystemMatrixType = typename BaseType::TSystemMatrixType;
+
+    using TSystemVectorType = typename BaseType::TSystemVectorType;
+
+    using LocalSystemVectorType = typename BaseType::LocalSystemVectorType;
+
+    using LocalSystemMatrixType = typename BaseType::LocalSystemMatrixType;
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    AlgebraicFluxCorrectedScalarSteadyScheme(const double RelaxationFactor, const Flags rBoundaryFlags)
+        : BaseType(),
+          mRelaxationFactor(RelaxationFactor),
+          mBoundaryFlags(rBoundaryFlags),
+          mrPeriodicIdVar(Variable<int>::StaticObject())
```

Comment:
```suggestion
    AlgebraicFluxCorrectedScalarSteadyScheme(
        const double RelaxationFactor,
        const Flags rBoundaryFlags)
        : BaseType(),
          mRelaxationFactor(RelaxationFactor),
          mBoundaryFlags(rBoundaryFlags),
          mrPeriodicIdVar(Variable<int>::StaticObject())
```
Style guide in new files.

Reply:
done :)